docs/experimental/functions.md

```diff
@@ -230,6 +281,44 @@ apis:
       followerSample: followers>follower.id
 ```
 
+
+## save_output
+
+| Applies to  | Description |
+| :---------- | :---------- |
+| API | Saves sample output to a .JSON file specified by the user, any directories in the path must exist prior. |
```

Comment:
Rewrite description to:

 Saves sample output to an existing directory as JSON.